src/System.Private.TypeLoader/src/Internal/Runtime/TypeLoader/ExternalReferencesTable.cs

```diff
@@ -161,8 +151,13 @@ unsafe public IntPtr GetFieldAddressFromIndex(uint index)
                 throw new BadImageFormatException();
 
             // TODO: indirection through IAT
-            int* pRelPtr32 = &((int*)_elements)[index];
```

Comment:
It may be nice to submit the supporting self-contained fixes like this one as separate PRs.`

Reply:
Do you think it's better to submit changes in `ExternalReferencesTable` and `ExternalReferencesTableNode` as separate PR?

Reply:
Up to you ... splitting large change into multiple smaller PRs that make sense on its own makes it  easier to push the changes through.